challenges/how-to-compute-absolute-value/compute-absolute-value-2.py

```diff
@@ -0,0 +1,11 @@
+# Link to the challenge: https://therenegadecoder.com/code/how-to-compute-absolute-value-in-python/#challenge
+# This solution using simple string methods
+
+number = input("Enter a number >>> ")
+sign = number[0] # It's obviuos that the sign(+ or -) will be at the beginning
+value = number[1:]
+
+if sign == '+':
+    print('+' * value)
```

Comment:
I don't think this works if the value is never cast to an int. Otherwise, this looks good!

Reply:
Ow, you mean if the input is `5` without the sign?

Reply:
I tried something like this, and it's not running for me:

```python
>>> number = input("Enter a number >>> ")
Enter a number >>> -7
>>> value = number[1:]
>>> '+' * value
```

Maybe I'm doing something wrong.

Reply:
I see. The problem is being caused because I forgot the turn the value into int. Fixing right now.